app/Services/PolygonService.php

```diff
@@ -83,7 +83,6 @@ public function createCriteriaSite($polygonId, $criteriaId, $valid): bool|string
         $criteriaSite->polygon_id = $polygonId;
         $criteriaSite->criteria_id = $criteriaId;
         $criteriaSite->valid = $valid;
-        $criteriaSite->created_by = Auth::user()?->id;
```

Comment:
Why was this removed?

Reply:
It was an error when testing some features in our enviroment. criteria_site doesn't have that column.

Reply:
Oh 🤦🏻 😂